src/elements/table/Table.js

```diff
@@ -259,8 +259,8 @@ export class NovoTable {
      * @param filter
      */
     onFilterClick(column, filter) {
-        if (Array.isArray(column.filter)) {
-            if (column.filter.includes(filter)) {
+        if (Array.isArray(column.filter) && column.multiple) {
```

Comment:
Not to pile on, but whatever the decision for multiple by default or single, can we call that out in the readme? 

https://github.com/bullhorn/novo-elements/blob/master/src/elements/table/README.md